content/source/docs/cloud/sentinel/import/tfplan-v2.html.md

```diff
@@ -333,6 +333,11 @@ An element contains the following fields:
   allows the provider to be interpreted unambiguously in the unusual situation
   where a provider offers a resource type whose name does not start with its own
   name, such as the `googlebeta` provider offering `google_compute_instance`.
+
+-> Starting with Terraform 0.13, the `provider_name` field will contain the
+_full_ source address to the provider in the Terraform Registry. Example:
+`registry.terraform.io/hashicorp/null` for the null provider.
```

Comment:
```suggestion
    -> **Note:** Starting with Terraform 0.13, the `provider_name` field contains the
    _full_ source address to the provider in the Terraform Registry. Example:
    `registry.terraform.io/hashicorp/null` for the null provider.
```

- Adding the conventional "**Note:**" prefix we use for blue callout boxes. 
- Present tense, since 0.13 is _on the loose._
- Fixing formatting: indenting by exactly four spaces will keep the note associated with its parent list item, instead of breaking this into two separate unordered lists with a paragraph between them.